### PR TITLE
print error information, when the os is not supported

### PR DESCRIPTION
**Pull request by knightXun**

### CODE REVIEW

File: tensorflow/go/test.sh [hunk 1]
```diff
@@ -63,6 +63,9 @@ then
   else
     export DYLD_LIBRARY_PATH="${PWD}/tensorflow:${DYLD_LIBRARY_PATH}"
   fi
+else 
+  echo "Only support Linux/Darwin.System $OS is not supported"
```
Comment (asimshankar) [inline tensorflow/go/test.sh]: Nit: Add a space between. Or make this:

```sh
echo "The test script works only on Linux or Darwin, not $OS."
```

**Review by asimshankar — CHANGES_REQUESTED** [review]
Thanks!

**Comment by knightXun** [comment]
＠asimshankar I think that's ok

**Review by asimshankar — APPROVED** [review]
Thanks!

**Pull request merged**